doc/setup_procedure.md

```diff
@@ -0,0 +1,84 @@
+# Setup Procedure
+
+Person 1 and person 2 work in parallel
+
+## Person 1 (Ciaran or Orion)
+
+1. Open the Deck box on table in front of the pilot.
+
+1. Look through the Lexan sheet on the deck box and check for any connectors that have fallen out.
+
+    * [wiring diagram](example.com)
+
+1. Screw the wifi antennas into their RP-SMA jacks on the deck box.
+
+1. Turn on the keyboard with the switch on the top edge.
+
+1. Plug the monitor into its power supply (USB-C cable) on the deck box.
+
+1. If WAN is available, connect it to the WAN port on the deck box.
+
+1. Plug the IEC-C13 Cable into the 120V power supplied by MATE.
+
+1. Plug the IEC-C13 cable into the IEC-C14 "POWER IN" port on the deck box.
+
+1. Flip the "POWER IN" switch to the ON position on the deck box.
+
+## Person 2 (Tether Manager)
+
+1. Connect the tether to the ROV.
+    * Ethernet
+    * Pneumatic Line
+    * Power Connector
+
+1. Connect the tether SBS50 to the 48v PSU
+
+1. Connect the tether Ethernet to the "ROV LAN" port on the deck box
+
+1. Connect the tether pneumatic line to the compressor
+
+1. Plug the IEC-C19 cable for the 48V PSU into the 120V power supplied by mate
+
+1. Plug the IEC-C19 cable into the IEC-C20 socket on the 48V PSU
+
+1. Flip the "INPUT SW" to "I" on the 48V PSU
+
+1. Flip the "OUTPUT SW" to "ON" on the 48V PSU
+
+1. Uncoil the XBOX Controller cable.
+
+1. Plug the XBOX Controller into "REAR USB 2"
+
+## Person 1 (Ciaran or Orion) (part 2)
+
+1. Wait for the monitor to show the Ubuntu Desktop.
+
+1. On the Keyboard use the `ctrl+alt+t` keyboard shortcut to open a Terminal window
```

Comment:
I'm working on automating this part.

Reply:
mike said we should explicitely not automate this because he wants to be able to see the console output if something crashes

Reply:
automating `gnome-terminal` launch with the commands running, persisting after they close into `bash`